### PR TITLE
Lpd 21835 - ERC on user api - contacts

### PR DESCRIPTION
**Pull request by matyaswollner**
https://liferay.atlassian.net/browse/LPD-21835

### CODE REVIEW

File: modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/EmailAddressResourceTest.java [hunk 1]
```diff
@@ -125,6 +125,24 @@ protected EmailAddress testGetEmailAddress_addEmailAddress()
 			ListTypeConstants.CONTACT_EMAIL_ADDRESS);
 	}
 
+	protected EmailAddress
+			testGetOrganizationByExternalReferenceCodeEmailAddressesPage_addEmailAddress(
+				String externalReferenceCode, EmailAddress emailAddress)
+		throws Exception {
+
+		return _addEmailAddress(
+			emailAddress, _organization.getModelClassName(),
```
Comment (smottal) [inline modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/EmailAddressResourceTest.java:134]: please use Organization.class.getName() - check if in the same by id we have to change it too

File: modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/PhoneResourceTest.java [hunk 2]
```diff
@@ -114,6 +114,26 @@ protected Long testGetAccountPhonesPage_getAccountId() throws Exception {
...
 	}
 
+	@Override
+	protected Phone
+			testGetOrganizationByExternalReferenceCodePhonesPage_addPhone(
+				String externalReferenceCode, Phone phone)
+		throws Exception {
+
+		return _addPhone(
+			phone, _organization.getModelClassName(),
```
Comment (smottal) [inline modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/PhoneResourceTest.java:124]: please use Organization.class.getName() and check if we need to update also the byId method

File: modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/PostalAddressResourceTest.java [hunk 3]
```diff
@@ -136,6 +156,26 @@ protected Long testGetAccountPostalAddressesPage_getAccountId()
...
 	}
 
+	@Override
+	protected PostalAddress
+			testGetOrganizationByExternalReferenceCodePostalAddressesPage_addPostalAddress(
+				String externalReferenceCode, PostalAddress postalAddress)
+		throws Exception {
+
+		return _addPostalAddress(
+			postalAddress, _organization.getModelClassName(),
```
Comment (smottal) [inline modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/PostalAddressResourceTest.java:166]: please use Organization.class.getName() and check if we need to update also the byId method

File: modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/WebUrlResourceTest.java [hunk 4]
```diff
@@ -113,6 +113,26 @@ protected Long testGetAccountWebUrlsPage_getAccountId() throws Exception {
...
 	}
 
+	@Override
+	protected WebUrl
+			testGetOrganizationByExternalReferenceCodeWebUrlsPage_addWebUrl(
+				String externalReferenceCode, WebUrl webUrl)
+		throws Exception {
+
+		return _addWebUrl(
+			webUrl, _organization.getModelClassName(),
```
Comment (smottal) [inline modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/WebUrlResourceTest.java:123]: please use Organization.class.getName() and check if we need to update also the byId method

**Review by smottal — CHANGES_REQUESTED** [review]
@matyaswollner just very little changes (for consistence, not for logic)

**Comment by matyaswollner** [comment]
ci:test:sf

**Review by smottal — APPROVED** [review]

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:stop

**Comment by smottal** [comment]
ci:forward:force

**Pull request closed without merging**